data-prepper-plugins/s3-source/README.md

```diff
@@ -22,6 +22,8 @@ The `compression` property defines how to handle compressed S3 objects. It has t
 
 * `none` - The file is not compressed.
 * `gzip` - Apply GZip de-compression on the S3 object.
+* `.snappy` - Apply Snappy de-compression on the S3 object.
```

Comment:
There is only one new option - `snappy`. I think combine these lines into:

```
* `snappy` - Apply Snappy decompression on the S3 object.
```

Reply:
Thanks.. updated to only one option

Reply:
Looks like we have '.' as prefix. This should just be `snappy`

Reply:
Thanks for the catch- fixed it